cli/flox/doc/flox-activate.md

```diff
@@ -80,10 +80,26 @@ hook was defined.
 
 # ENVIRONMENT VARIABLES
 
-## `$_FLOX_ACTIVE_ENVIRONMENTS`
-A JSON array containing one object per active environment.
-This is currently an implementation detail
-and its contents are subject to change.
+`$_FLOX_ACTIVE_ENVIRONMENTS`
+:   A JSON array containing one object per active environment.
+    This is currently an implementation detail
+    and its contents are subject to change.
+
+`$_FLOX_ACTIVE_ENVIRONMENTS`
```

Comment:
suggestion: I think this is supposed to be `FLOX_PROMPT_ENVIRONMENTS`

Reply:
Correct, copy and paste got me